tooling/gatsby-plugin/src/provider.js

```diff
@@ -0,0 +1,21 @@
+import * as React from "react"
+import { ChakraProvider, extendTheme } from "@chakra-ui/react"
+import userTheme from "./theme"
+
+const theme = extendTheme(userTheme)
+
+export const WrapRootElement = ({
```

Comment:
The `isUsingColorMode` is obsolete really :/

Reply:
I don't mind a breaking change for this plugin 🤷  what do you think @chakra-ui/react-team

Reply:
Yeah I'm totally fine with that!